resources/views/device/tabs/notes.blade.php

```diff
@@ -0,0 +1,14 @@
+@extends('device.index')
+
+@section('tab')
+    <x-panel title="{{ __('Device Notes') }}">
```

Comment:
@lang() blade directive is preferred

Reply:
It doesn't work because `title` is a variable, so it returns `Call to undefined function lang()`

Reply:
Pick your poison:  
`title="@lang('Device Notes')"` = `@lang('Device Notes')`  
`title="{{ @lang('Device Notes') }}"` = `Call to undefined function lang()`  
`:title="@lang('Device Notes')"` = `Call to undefined function lang()`  
`title="{{ __('Device Notes') }}` = `Device Notes`

Reply:
Huh, components must be applied first.